docs/advanced-features/tabulator.md

```diff
@@ -188,3 +194,128 @@ if state == 'SUCCEEDED':
 else:
     print(f'Query did not succeed. Final state: {state}')
 ```
+
+## Unrestricted Access
+
+> Available since Quilt Platform version 1.57
+
+By default, Tabulator is only accessible via a session provided by the Quilt Catalog,
+and the access is scoped to the permissions of the Catalog user associated with
+that session. However, an admin can enable **unrestricted access** to Tabulator,
+deferring all access control to AWS. The underlying data in S3 is accessed using
+the Tabulator's dedicated "unrestricted" role, which has read-only access to all
+the S3 buckets attached to the given stack. This allows querying the data directly
+from the AWS Console or Athena views, given the caller has the necessary permissions
+to access Athena resources associated with Tabulator.
+
+![Tabulator Settings](../imgs/admin-tabulator-settings.png)
+
+### Permissions & Configuration
+
+In order to access Tabulator in unrestricted mode, the caller must:
+
+1. Provide a workgroup with output location set and compatible with that of the
+   Tabulator (`s3://${UserAthenaResultsBucket}/athena-results/non-managed-roles/`).
+
+2. Have the following permissions:
+
+   - Athena query execution on the designated workgroup
+   - Access to the Tabulator data catalog
+   - Invoking the Tabulator Lambda function
+   - Read access to the Tabulator bucket for spill files
+   - Read/write access to the Athena results bucket
+
+Here is an example CloudFormation template that creates the necessary resources:
+
+```yaml
+AWSTemplateFormatVersion: 2010-09-09
+Description: "Resources for accessing Tabulator in unrestricted mode"
+
+Parameters:
```

Comment:
I apologize if this is a stupid question, but it seems like we already know all of this.  Could we create a Policy that encapsulates this, so they just have to add that to their Role?

Reply:
In particular, if they already have a Role with 'unrestricted' abilty to call Athena and Lambdas, then won't that automatically be able to call Tabulator? If not, then what exactly were we protecting against...

Reply:
Another thing that confuses me.  It looks like we now *always* use a split folder, and never return any results inline:
```
    fn should_spill(&self, _df: &DataFrame) -> bool {
        true
    }
```
If so, can't we just use that as the way to gate access, so we don't need restricted mode at all? Or am I just seriously confused?

Reply:
> Could we create a Policy that encapsulates this, so they just have to add that to their Role?

yeah we can. i was considering this, but for some reason decided to not follow thru. i think i'm reconsidering this -- i'll add both the policy and the workgroup to the template and export their ids/arns for easier consumption


> In particular, if they already have a Role with 'unrestricted' abilty to call Athena and Lambdas, then won't that automatically be able to call Tabulator? If not, then what exactly were we protecting against...

call tabulator -- sure, but the tabulator itself uses two roles for functioning:
1. its "own" role with access to cache and stuff
2. the role it gets from the registry and assumes to actually access the data (in unrestricted mode it's the special "unrestricted" role, otherwise it's a catalog user's role)

so even if someone is calling tabulator without the proper setup it will just crash -- return an error, not the data

> Another thing that confuses me. It looks like we now always use a split folder, and never return any results inline:

correct

> If so, can't we just use that as the way to gate access, so we don't need restricted mode at all? Or am I just seriously confused?

maybe we could, but that removes an extra layer of security, so i don't think we should do that